pkg/services/encryption/ossencryption/ossencryption.go

```diff
@@ -0,0 +1,88 @@
+package ossencryption
```

Comment:
I'm wondering if we should rename to something else like just `service` or `encryptionservice` cause `oss` sounds redundant within an open source code base 🤭  Though we already have some "OSS" references as part of other packages names / types.

What do you think? @sakjur 

Thanks!! :)

Reply:
I'm not fond of `OSS` in package names either and try to not use it myself, but I don't have a better alternative in mind.

But yeah, we've done it elsewhere 🤷